plugins/ajax/plugin.js

```diff
@@ -65,14 +65,25 @@
 			return null;
 		}
 
-		function load( url, callback, getResponseFn ) {
+		function getCustomResponse( xhr ) {
```

Comment:
Won't it be better to create one `getResponse` function that would return appropriate response instead of creating one function for every response type? Especially that now `ArrayBuffer` is supported only in `load` method and having one function would enable it everywhere with minimal amount of work.

If not, then at least change the function name to something more appropriate, e.g. `getResponseArrayBuffer`.